server/data/vocabularies.json

```diff
@@ -251,5 +251,31 @@
         "service": {"all": 1},
         "init_version": 1,
         "items": []
+    },
+    {
+        "_id": "importance",
```

Comment:
it would be better to match the `scheme`, actually in general we don't put the scheme in the items, it needs to be there only when using the items and adding it to the article

Reply:
so this should be actually `stturgency`